Sofa/framework/Helper/src/sofa/helper/system/PluginManager.cpp

```diff
@@ -255,11 +266,25 @@ void PluginManager::addOnPluginLoadedCallback(const std::string& key, std::funct
     }
 }
 
+void PluginManager::addOnPluginCleanupCallbacks(const std::string& key, std::function<void()> callback)
+{
+    if(m_onPluginCleanupCallbacks.find(key) == m_onPluginCleanupCallbacks.end())
+    {
+        m_onPluginCleanupCallbacks[key] = callback;
+    }
```

Comment:
replace with std::map::insert as it requires only one instruction

Reply:
Ok, I've also cleaned the one I've copied from.